src/main/java/org/pac4j/vertx/handler/impl/CallbackDeployingPac4jAuthHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.pac4j.vertx.handler.impl;
+
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.pac4j.core.config.Config;
+import org.pac4j.vertx.auth.Pac4jAuthProvider;
+
+/**
+ * @author jez
+ */
+public class CallbackDeployingPac4jAuthHandler extends RequiresAuthenticationHandler {
```

Comment:
I think I already told you, but I'd like READMEs to be as close as possible between implementations: see https://github.com/pac4j/j2e-pac4j/blob/master/README.md and https://github.com/pac4j/play-pac4j/blob/master/README.md. Any of them is a good template. Note they may change a little before the final release of the *-pac4j libraries (in two weeks or so).